pkg/iss/leaderselectionpolicy.go

```diff
@@ -21,6 +21,13 @@ type LeaderSelectionPolicy interface {
 
 	// Suspect updates the state of the policy object by announcing it that node `node` has been suspected in epoch `e`.
 	Suspect(e t.EpochNr, node t.NodeID)
+
+	// Reconfigure returns a new LeaderSelectionPolicy based on the state of the current one,
+	// but using a new configuration.
+	// TODO: Use the whole configuration, not just the membership.
+	Reconfigure(membership []t.NodeID) LeaderSelectionPolicy
```

Comment:
the same comment about membership for this and all other places.  Maybe nodeIDs? It sounds better with "Reconfigure"

Reply:
Fixed.